packages/engine/Source/Scene/Fog.js

```diff
@@ -32,8 +40,41 @@ function Fog() {
    * Decreasing the value will push the fog further from the viewer, but decrease performance as more of the terrain is rendered.
    * @type {number}
    * @default 2.0e-4
+   * @example
+   * // Double the default fog density
+   * viewer.scene.fog.density = 0.0004;
    */
   this.density = 2.0e-4;
+  /**
+   * A scalar used in the function to adjust density based on the height of the camera above the terrain.
+   * @type {number}
+   * @default 0.001
+   */
+  this.fogScalar = 0.001;
```

Comment:
```suggestion
  this.heightScalar = 0.001;
```

Fog should be implicit since this is the `Fog` class... but height may not be.